apps/src/blockly/addons/cdoXml.ts

```diff
@@ -10,39 +12,33 @@ import {
 // Once this has been done, all subsequent steps in the serialization use userCreated.
 const USER_CREATED_XML_ATTRIBUTE = 'usercreated';
 
-export default function initializeBlocklyXml(blocklyWrapper) {
-  // Clear xml namespace
-  blocklyWrapper.utils.xml.NAME_SPACE = '';
+export default function initializeBlocklyXml(
+  blocklyWrapper: BlocklyWrapperType
+) {
+  // Clear xml namespace. This property is readonly in Google Blockly.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (blocklyWrapper.utils.xml as any).NAME_SPACE = '';
 
   // Aliasing Google's domToBlock() so that we can override it, but still be able
   // to call Google's domToBlock() in the override function.
-  blocklyWrapper.Xml.originalDomToBlock = blocklyWrapper.Xml.domToBlock;
```

Comment:
there wasn't a reason to assign this to the object since we only use it below, so I updated this to just be a constant.